### PR TITLE
Istio: Download istioctl binary from gcp mirror

### PR DESCRIPTION
**Pull request by rhrazdil**
Signed-off-by: Radim Hrazdil <rhrazdil@redhat.com>

### CODE REVIEW

**Review by fgimenez — APPROVED** [review]
/approve
/hold

Thanks! Feel free to unhold when ready.

File: cluster-provision/k8s/1.21/provision.sh [hunk 1]
```diff
@@ -86,11 +87,12 @@ dnf -y install lvm2
...
 
 # Install istioctl
-export PATH=/opt/istio-$ISTIO_VERSION/bin:$PATH
+export PATH=$ISTIO_BIN_DIR:$PATH
 (
   set -E
-  cd /opt/
-  curl -L https://istio.io/downloadIstio | sh -
+  mkdir -p $ISTIO_BIN_DIR
+  curl https://storage.googleapis.com/kubevirtci-istioctl-mirror/istio-$ISTIO_VERSION/bin/istioctl -o $ISTIO_BIN_DIR/istioctl
```
[the remark this answers is not in the document]
Comment (phoracek) [reply cluster-provision/k8s/1.21/provision.sh:94]: I believe that periodic mirroring is handled via https://github.com/kubevirt/project-infra/pull/1378

[the remark this answers is not in the document]
Comment (rhrazdil) [reply cluster-provision/k8s/1.21/provision.sh:94]: Correct, the mirroring is done by the [periodic](https://github.com/kubevirt/project-infra/pull/1378).
I might think of automating bumping of the Istio version in a followup though

**Pull request merged**